pkgs/development/python-modules/python-sat/default.nix

```diff
@@ -0,0 +1,22 @@
+{ buildPythonPackage, fetchPypi, lib, six, pypblib, pytest }:
+
+buildPythonPackage rec {
+	pname = "python-sat";
+  version = "0.1.6.dev6";
+
+  src = fetchPypi {
+		inherit pname version;
+    sha256 = "7v8IP0JK3Xjmf4uuY/Z25aRINU15A2H0hNGFI8E5Or0=";
+  };
+
+	propagatedBuildInputs = [ six pypblib ];
+
+	doCheck = false; # test are interactive
```

Comment:
If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

Reply:
tests weren't included on pypi, but they were on github